docs/usage/modules/versioning.md

```diff
@@ -22,7 +52,18 @@ Configuring or overriding the default `versioning` can be helpful for ecosystems
 }
 ```
```

Comment:
Sorry I think this is now a duplicate of the one I added earlier

Reply:
Yeah we have a duplicate now. 😄

Reply:
What heading do you like best:

- `Docker versioning`
- `Overriding Docker versioning to use a versioning specific for a package`

Reply:
I've picked the more descriptive title to keep.